priv/repo/migrations/20210924114509_add_post_history_on_posts.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Erlef.Repo.Migrations.AddPostHistoryOnPosts do
```

Comment:
We should just collapse this into the main migration since it's part of the same PR.